### DISCUSSION TITLE
Uncaught TypeError: can't access property "call", Transform is undefined

### DISCUSSION THREAD

**Discussion by TusharPardhe** [webpack/webpack#16133] [General]
# Bug report

**What is the current behavior?**
Using xrpl.js library and trying to import a function but when I use webpack + react it doesnt work, however, its working normally in nodejs without webpack.

**If the current behavior is a bug, please provide the steps to reproduce.**

Steps to reproduce:
1. Import xrpl library 
`import { Wallet } from "xrpl";
console.log(Wallet.fromSeed("sstw2w8CxZYeCDmSUeStc5SbqPAQr"));
`
2. use this method and you might get this error.

My webpack config:
`const path = require("path");
const { CleanWebpackPlugin } = require("clean-webpack-plugin");
const HtmlWebpackPlugin = require("html-webpack-plugin");
const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
const CopyWebpackPlugin = require("copy-webpack-plugin");
const Dotenv = require('dotenv-webpack');

module.exports = {
    entry: path.resolve(__dirname, "..", "./src/index.js"),
    // target: 'node',
    module: {
        rules: [
            {
                test: /\.(js|jsx)$/,
                exclude: [/node_modules/],
                use: ["babel-loader"],
            },
            {
                test: /\.(jpg|png|gif|jpeg|woff|woff2|eot|ttf|svg)$/,
                use: ["url-loader?limit=100000"],
            },
        ],
    },
    resolve: {
        extensions: ["*", ".js", ".jsx"],
        fallback: {
            "fs": false
        },
    },
    plugins: [
        new CleanWebpackPlugin(),
        new HtmlWebpackPlugin({
            template: path.resolve(__dirname, "..", "./src/index.html"),
            favicon: "./favicon.ico",
            minify: {
                removeComments: true,
                collapseWhitespace: true
            }
        }),
        new NodePolyfillPlugin(),
        new CopyWebpackPlugin({
            patterns: [
                { from: path.resolve(__dirname, "..", "manifest.json") },
                { from: path.resolve(__dirname, "..", "xpt192.png") },
                { from: path.resolve(__dirname, "../src/scripts/background.js") },
                { from: path.resolve(__dirname, "../src/scripts/content.js") },
            ],
        }),
        new Dotenv({
            path: path.resolve(__dirname, "../.env")
        }),
    ],
    output: {
        path: path.resolve(__dirname, "..", "dist"),
        filename: "bundle.js",
        publicPath: "/",
    },
    devServer: {
        historyApiFallback: true,
        inline: true,
    },
};`

**What is the expected behavior?**
Import should work

**Other relevant information:**
webpack version:  "^5.72.1"
Node.js version: 16
Operating System: mac
Additional tools:

**Comment by alexander-akait** [comment]
Please create reproducible test repo
